library.py

```diff
@@ -31,6 +31,9 @@
 mail = Mail(app)
 bootstrap = Bootstrap(app)
 
+def logout_user():
+    session.pop('logged_in', None)
+
 def login_required(test):
```

Comment:
Thought a function would be easier to remember and there's two places the code is logging out now.